src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentProjectsFinder.cs

```diff
@@ -299,7 +299,7 @@ private static bool IsInternalsVisibleToAttribute(AttributeData attr)
 
             var internalsVisibleToMap = CreateInternalsVisibleToMap(sourceAssembly);
 
-            SymbolKey? sourceAssemblySymbolKey = null;
+            var sourceAssemblySymbolKey = sourceAssembly.GetSymbolKey();
```

Comment:
these symbol keys are not expensive to make.  I'm just removing this premature optimization where we attempt to delay create these.